### ISSUE TITLE
Failure to start （version DBeaver 23.3.3.202401211839）

### ISSUE THREAD

**Issue by zhonghuazhi** [dbeaver/dbeaver#22890]
### Description

The software cannot start properly

### DBeaver Version

DBeaver 23.3.3.202401211839

### Operating System

macOS 14.3.1 (23D60)

### Additional context

2024-02-20 16:43:34.191 - DBeaver 23.3.3.202401211839 is starting
2024-02-20 16:43:34.192 - OS: Mac OS X 14.3.1 (x86_64)
2024-02-20 16:43:34.192 - Java version: 17.0.6 by Eclipse Adoptium (64bit)
2024-02-20 16:43:34.192 - Install path: '/Applications/DBeaver.app/Contents/Eclipse'
2024-02-20 16:43:34.192 - Instance path: 'file:/Users/zhonghua.zhi/Library/DBeaverData/workspace6/'
2024-02-20 16:43:34.194 - Memory available 140Mb/1024Mb
2024-02-20 16:43:34.203 - Create display
2024-02-20 16:43:34.376 - Initialize desktop platform...
2024-02-20 16:43:34.490 - BounceCastle bundle found. Use JCE provider BC
> Start Core File Systems [org.eclipse.core.filesystem 1.10.200.v20231102-0934]
> Start Core Resource Management [org.eclipse.core.resources 3.20.0.v20231102-0934]
> Start 小组支持核心 [org.eclipse.team.core 3.10.200.v20231106-1240]
> Start DBeaver UI Navigator [org.jkiss.dbeaver.ui.navigator 1.0.123.202401211839]
2024-02-20 16:43:34.889 - Initialize base platform...
2024-02-20 16:43:34.908 - Platform initialized (532ms)
> Start DBeaver UI Editors - Base [org.jkiss.dbeaver.ui.editors.base 1.0.123.202401211839]
2024-02-20 16:43:35.529 - Starting instance server at http://localhost:61226
2024-02-20 16:43:35.535 - Run workbench
> Start Eclipse e4 工作台 SWT [org.eclipse.e4.ui.workbench.swt 0.17.200.v20231030-2045]
> Start Eclipse Application Services [org.eclipse.e4.core.services 2.4.200.v20231103-2012]
> Start Eclipse UI Application Services [org.eclipse.e4.ui.services 1.6.200.v20231030-2045]
> Start Eclipse e4 Progress View [org.eclipse.e4.ui.progress 0.4.200.v20231030-2045]
> Start Eclipse Bindings Support [org.eclipse.e4.ui.bindings 0.14.200.v20231030-2045]
> Start Eclipse e4 core commands [org.eclipse.e4.core.commands 1.1.200.v20231108-1058]
> Start EMF XML/XMI Persistence [org.eclipse.emf.ecore.xmi 2.36.0.v20231002-1156]
> Start Eclipse e4 工作台加载项 [org.eclipse.e4.ui.workbench.addons.swt 1.5.200.v20231106-1600]
> Start Eclipse e4 工作台 SWT 呈现器 [org.eclipse.e4.ui.workbench.renderers.swt 0.16.200.v20231030-2045]
> Start Eclipse Dependency Injection Extensions [org.eclipse.e4.core.di.extensions 0.18.100.v20231103-2012]
> Start Eclipse CSS SWT Support [org.eclipse.e4.ui.css.swt 0.15.200.v20231025-0632]
Warning: NLS unused message: BlockedJobsDialog_UserInterfaceTreeElement in: org.eclipse.ui.internal.progress.messages
> Start Eclipse CSS SWT Theme Support [org.eclipse.e4.ui.css.swt.theme 0.14.200.v20230913-1724]
2024-02-20 16:43:37.133 - Configure workbench window
> Start DBeaver SQL Model [org.jkiss.dbeaver.model.sql 1.0.113.202401211839]
2024-02-20 16:43:37.251 - Total database drivers: 107 (107)
> Start 帮助系统核心 [org.eclipse.help 3.10.200.v20231106-1240]
2024-02-20 16:43:37.305 - Create actions
2024-02-20 16:43:37.311 - Disable Eclipse action set 'org.eclipse.ui.edit.text.actionSet.annotationNavigation'
2024-02-20 16:43:37.311 - Disable Eclipse action set 'org.eclipse.ui.WorkingSetActionSet'
2024-02-20 16:43:37.311 - Disable Eclipse action set 'org.eclipse.ui.actionSet.openFiles'
Warning: NLS unused message: pref_page_ui_general_group_timezone in: org.jkiss.dbeaver.core.CoreResources
> Start jna [com.sun.jna 5.13.0.v20230812-1000]
2024-02-20 16:43:37.698 - Initialize workbench window
2024-02-20 16:43:37.702 - Finish initialization
> Start DBeaver UI Editors - Data [org.jkiss.dbeaver.ui.editors.data 1.0.123.202401211839]
> Start GEF Classic Draw2d [org.eclipse.draw2d 3.14.100.202311222042]
> Start GEF Classic GEF (MVC) [org.eclipse.gef 3.16.0.202311222042]
> Start DBeaver ERD UI [org.jkiss.dbeaver.erd.ui 3.0.83.202401211839]
> Start 因特网连接管理 UI [org.eclipse.ui.net 1.5.200.v20231106-1240]
> Start Internet Connection Management [org.eclipse.core.net 1.5.200.v20231106-1240]
Warning: NLS unused message: pref_page_sql_editor_group_misc in: org.jkiss.dbeaver.ui.editors.sql.internal.SQLEditorMessages
> Start DBeaver UI Editors - SQL [org.jkiss.dbeaver.ui.editors.sql 1.0.123.202401211839]
> Start DBeaver Sample Database [org.jkiss.dbeaver.ext.sample.database 1.0.160.202401211839]
> Start DBeaver Tip of the day [org.jkiss.dbeaver.ext.ui.tipoftheday 1.0.129.202401211839]
> Start DBeaver ERD Model [org.jkiss.dbeaver.erd.model 1.0.83.202401211839]
> Start 缺省文本编辑器 [org.eclipse.ui.editors 3.17.100.v20231106-1600]
> Start 文本编辑器框架 [org.eclipse.ui.workbench.texteditor 3.17.200.v20231106-1600]
2024-02-20 16:43:41.168 - Open [https://dbeaver.io/product/version.xml]
2024-02-20 16:46:49.388 - DBeaver is stopping
2024-02-20 16:46:49.388 - Stop instance server
2024-02-20 16:46:50.403 - Instance server has been stopped
2024-02-20 16:46:50.414 - DBeaver shutdown completed

**Comment by E1izabeth** [comment]
Please, try latest Dveaver version

**Comment by zhonghuazhi** [comment]
The latest version is also unavailable

**Comment by E1izabeth** [comment]
Could you describe how is it happening? I don't see any unusual record in the logs. Do you see only splash screen or DBeaver starts working and then unexpectedly is being closed?

**Comment by serge-rider** [comment]
You also could try to reset UI settings: https://github.com/dbeaver/dbeaver/wiki/Reset-UI-settings

**Comment by eride** [comment]
I had a similar issue where DBeaver was only loading up to the splash screen but wouldn't go any further and was permanently stuck there.

The cause of the issue was multiple older configs in the "Old Default" or "Ancient Default" locations: https://dbeaver.com/docs/dbeaver/Workspace-Location/#old-default-workspace-location

In addition to these, I also had one called `~/.local/share/.DBeaverData` (with a `.` in front of DBeaverData), so in your case you might want to check for a `~/Library/.DBeaverData/`, `~/.dbeaver/`, `~/.dbeaver4/`

What worked for me was renaming all the config dirs to .dbeaver4_OLD / DBeaverData_OLD / .DBeaverData_OLD so that DBeaver could run and I could figure out which config directory was correct, then copying the good configs over to the new directory.

**Comment by zhonghuazhi** [comment]
> You also could try to reset UI settings: https://github.com/dbeaver/dbeaver/wiki/Reset-UI-settings

The operation tried, still no effect

**Comment by zhonghuazhi** [comment]
> Could you describe how is it happening? I don't see any unusual record in the logs. Do you see only splash screen or DBeaver starts working and then unexpectedly is being closed?

When the dbeaver software is running normally, the system prompts you to upgrade dbeaver. After the upgrade is complete, the operating system is restarted, and dbeaver cannot be started again

**Comment by zhonghuazhi** [comment]
> `~/Library/.DBeaverData`



> I had a similar issue where DBeaver was only loading up to the splash screen but wouldn't go any further and was permanently stuck there.
> 
> The cause of the issue was multiple older configs in the "Old Default" or "Ancient Default" locations: https://dbeaver.com/docs/dbeaver/Workspace-Location/#old-default-workspace-location
> 
> In addition to these, I also had one called `~/.local/share/.DBeaverData` (with a `.` in front of DBeaverData), so in your case you might want to check for a `~/Library/.DBeaverData/`, `~/.dbeaver/`, `~/.dbeaver4/`
> 
> What worked for me was renaming all the config dirs to .dbeaver4_OLD / DBeaverData_OLD / .DBeaverData_OLD so that DBeaver could run and I could figure out which config directory was correct, then copying the good configs over to the new directory.

The following directory does not exist

~/.local/share/.DBeaverData

~/Library/.DBeaverData

**Comment by zhonghuazhi** [comment]
> You also could try to reset UI settings: https://github.com/dbeaver/dbeaver/wiki/Reset-UI-settings

Trying to uninstall dbeaver, reinstallation still has problems

**Comment by serge-rider** [comment]
> The following directory does not exist

Log says that your workspace is in `/Users/zhonghua.zhi/Library/DBeaverData/`
So you could try to delete folder `/Users/zhonghua.zhi/Library/DBeaverData/workspace6/.metadata/.plugins`

**Comment by zhonghuazhi** [comment]
> > The following directory does not exist
> 
> Log says that your workspace is in `/Users/zhonghua.zhi/Library/DBeaverData/` So you could try to delete folder `/Users/zhonghua.zhi/Library/DBeaverData/workspace6/.metadata/.plugins`

[Directory presence]
/Users/zhonghua.zhi/Library/DBeaverData/workspace6/.metadata    


but  .plugins  Directory does not exist

dbeaver Software fails to start

**Comment by E1izabeth** [comment]
Could you try uninstalling DBeaver, then downloading dbeaver installation from the web site, and installing again and chosing Reset settings in the installation wizard?
![image](https://github.com/dbeaver/dbeaver/assets/28875055/1e3de58d-04f1-4199-a79e-5eb57ab20324)

**Comment by jonathan-improvado** [comment]
I have the same problem  in Fedora 39.
It was working perfectly, but after shutting it down yesterday. Today it just doesn't open anymore. I had version 24.0 initially, but after reinstalling and continuing to share, I installed version 23.3.3 (via rpm).

These are the logs
```
2024-03-08 11:38:45.016 - DBeaver 23.3.3.202401211839 is starting
2024-03-08 11:38:45.017 - OS: Linux 6.7.7-200.fc39.x86_64 (amd64)
2024-03-08 11:38:45.017 - Java version: 17.0.6 by Eclipse Adoptium (64bit)
2024-03-08 11:38:45.017 - Install path: '/usr/share/dbeaver-ce'
2024-03-08 11:38:45.017 - Instance path: 'file:/home/jony/.local/share/DBeaverData/workspace6/'
2024-03-08 11:38:45.019 - Memory available 64Mb/1024Mb
2024-03-08 11:38:45.026 - Create display
2024-03-08 11:38:45.123 - Initialize desktop platform...
2024-03-08 11:38:45.214 - BounceCastle bundle found. Use JCE provider BC
> Start Core File Systems [org.eclipse.core.filesystem 1.10.200.v20231102-0934]
> Start Core Resource Management [org.eclipse.core.resources 3.20.0.v20231102-0934]
> Start Núcleo del equipo de soporte [org.eclipse.team.core 3.10.200.v20231106-1240]
> Start DBeaver UI Navigator [org.jkiss.dbeaver.ui.navigator 1.0.123.202401211839]
2024-03-08 11:38:45.375 - Initialize base platform...
2024-03-08 11:38:45.393 - Platform initialized (270ms)
> Start DBeaver UI Editors - Base [org.jkiss.dbeaver.ui.editors.base 1.0.123.202401211839]
2024-03-08 11:38:45.418 - Starting instance server at http://localhost:43757
2024-03-08 11:38:45.423 - Run workbench
> Start Eclipse e4 Workbench SWT [org.eclipse.e4.ui.workbench.swt 0.17.200.v20231030-2045]
> Start Eclipse Application Services [org.eclipse.e4.core.services 2.4.200.v20231103-2012]
> Start Eclipse UI Application Services [org.eclipse.e4.ui.services 1.6.200.v20231030-2045]
> Start Eclipse e4 Progress View [org.eclipse.e4.ui.progress 0.4.200.v20231030-2045]
> Start Eclipse Bindings Support [org.eclipse.e4.ui.bindings 0.14.200.v20231030-2045]
> Start Eclipse e4 core commands [org.eclipse.e4.core.commands 1.1.200.v20231108-1058]
> Start EMF XML/XMI Persistence [org.eclipse.emf.ecore.xmi 2.36.0.v20231002-1156]
> Start Eclipse CSS SWT Theme Support [org.eclipse.e4.ui.css.swt.theme 0.14.200.v20230913-1724]
> Start Eclipse e4 Workbench Add-ons [org.eclipse.e4.ui.workbench.addons.swt 1.5.200.v20231106-1600]
> Start Eclipse e4 Workbench SWT Renderer [org.eclipse.e4.ui.workbench.renderers.swt 0.16.200.v20231030-2045]
> Start Eclipse Dependency Injection Extensions [org.eclipse.e4.core.di.extensions 0.18.100.v20231103-2012]
> Start Eclipse CSS SWT Support [org.eclipse.e4.ui.css.swt 0.15.200.v20231025-0632]
Warning: NLS unused message: BlockedJobsDialog_UserInterfaceTreeElement in: org.eclipse.ui.internal.progress.messages
> Start Interfaz de usuario de manejo de conexión a Internet [org.eclipse.ui.net 1.5.200.v20231106-1240]
> Start Internet Connection Management [org.eclipse.core.net 1.5.200.v20231106-1240]
2024-03-08 11:38:46.624 - DBeaver is stopping
2024-03-08 11:38:46.627 - Stop instance server
2024-03-08 11:38:47.630 - Instance server has been stopped
2024-03-08 11:38:47.639 - DBeaver shutdown completed
```

I don't have any plugin installed.

If I run `dbeaver-ce` as a admin, the application starts without problem. 🤔

**Comment by serjiokov** [comment]
Please try to specify the path to the new workspace in "dbeaver.ini" file using the following format:
-data  "/Users/zhonghua.zhi/Library/DBeaverData/workspace7"

**Comment by E1izabeth** [comment]
It's been a while since no update here. If the issue is still actual please let me know and provide additional information.

**Comment by q2dm8** [comment]
@E1izabeth 

I have same issue. I have tried all the measures which I could find on the web
- delete dbeaver data
- deleted all the files connected with dbeaver and installed from scratch
- tried to run it as admin
- tried to clean workspace6 and plugins
And still am suffering from this issue. 

latest log:

2024-10-05 13:50:33.637 - DBeaver 24.2.1.202409221725 is starting
2024-10-05 13:50:33.638 - OS: Windows 10 10.0 (amd64)
2024-10-05 13:50:33.638 - Java version: 17.0.6 by Eclipse Adoptium (64bit)
2024-10-05 13:50:33.638 - Install path: 'C:\Program Files\DBeaver'
2024-10-05 13:50:33.638 - Instance path: 'file:/C:/Users/юзер/AppData/Roaming/DBeaverData/workspace6/'
2024-10-05 13:50:33.639 - Memory available 207Mb/1024Mb
2024-10-05 13:50:33.640 - Create display
2024-10-05 13:50:33.700 - Initialize desktop platform...
2024-10-05 13:50:33.769 - BounceCastle bundle found. Use JCE provider BC
> Start Core File Systems [org.eclipse.core.filesystem 1.11.0.v20240824-0952]
> Start Core Resource Management [org.eclipse.core.resources 3.21.0.v20240805-1607]
> Start Team Support Core [org.eclipse.team.core 3.10.500.v20240621-0541]
> Start DBeaver UI Navigator [org.jkiss.dbeaver.ui.navigator 1.0.139.202409221725]
2024-10-05 13:50:33.851 - 
2024-10-05 13:50:33.851 - 	Refreshing workspace.
2024-10-05 13:50:33.852 - Refreshing workspace contents
2024-10-05 13:50:33.857 - Error injecting field 'platformInstance' in 'org.jkiss.dbeaver.model.impl.app.ApplicationWorkbenchImpl@6badba10'
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.jkiss.dbeaver.utils.RuntimeUtils.injectComponentReferences(RuntimeUtils.java:708)
	at org.jkiss.dbeaver.utils.RuntimeUtils.getBundleService(RuntimeUtils.java:680)
	at org.jkiss.dbeaver.runtime.DBWorkbench.getApplicationWorkbench(DBWorkbench.java:45)
	at org.jkiss.dbeaver.runtime.DBWorkbench.getPlatform(DBWorkbench.java:51)
	at org.jkiss.dbeaver.registry.SWTBrowserRegistry.getActiveBrowser(SWTBrowserRegistry.java:77)
	at org.jkiss.dbeaver.registry.SWTBrowserRegistry.overrideBrowser(SWTBrowserRegistry.java:106)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.start(DBeaverApplication.java:262)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:208)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:143)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:109)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:439)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:271)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.invokeFramework(DBeaverLauncher.java:650)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.basicRun(DBeaverLauncher.java:610)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.run(DBeaverLauncher.java:1461)
Caused by: java.lang.NoClassDefFoundError: Could not initialize class com.sun.jna.Native
	at com.sun.jna.platform.win32.WinBase$WIN32_FIND_DATA.sizeOf(WinBase.java:755)
	at org.eclipse.core.internal.filesystem.local.Win32Handler.<clinit>(Win32Handler.java:196)
	at org.eclipse.core.internal.filesystem.local.LocalFileNativesManager.setUsingNative(LocalFileNativesManager.java:70)
	at org.eclipse.core.internal.filesystem.local.LocalFileNativesManager.reset(LocalFileNativesManager.java:52)
	at org.eclipse.core.internal.filesystem.local.LocalFileNativesManager.<clinit>(LocalFileNativesManager.java:45)
	at org.eclipse.core.internal.filesystem.local.LocalFile.fetchInfo(LocalFile.java:215)
	at org.eclipse.core.filesystem.provider.FileStore.fetchInfo(FileStore.java:231)
	at org.eclipse.core.internal.utils.FileUtil.realPath(FileUtil.java:134)
	at org.eclipse.core.internal.utils.FileUtil.realURI(FileUtil.java:181)
	at org.eclipse.core.internal.localstore.FileSystemResourceManager.setLocation(FileSystemResourceManager.java:1112)
	at org.eclipse.core.internal.localstore.FileSystemResourceManager.getStoreRoot(FileSystemResourceManager.java:624)
	at org.eclipse.core.internal.localstore.FileSystemResourceManager.getStoreRoot(FileSystemResourceManager.java:627)
	at org.eclipse.core.internal.localstore.FileSystemResourceManager.getStore(FileSystemResourceManager.java:581)
	at org.eclipse.core.internal.resources.Resource.getStore(Resource.java:1234)
	at org.eclipse.core.internal.resources.Project.assertCreateRequirements(Project.java:101)
	at org.eclipse.core.internal.resources.Project.create(Project.java:305)
	at org.eclipse.core.internal.resources.Project.create(Project.java:291)
	at org.eclipse.core.internal.resources.Project.create(Project.java:286)
	at org.jkiss.dbeaver.model.rcp.EclipseWorkspaceImpl.createDefaultProject(EclipseWorkspaceImpl.java:239)
	at org.jkiss.dbeaver.model.rcp.EclipseWorkspaceImpl.initializeProjects(EclipseWorkspaceImpl.java:93)
	at org.jkiss.dbeaver.core.DesktopPlatform.initialize(DesktopPlatform.java:141)
	... 23 more
> Start DBeaver UI Editors - Base [org.jkiss.dbeaver.ui.editors.base 1.0.139.202409221725]
2024-10-05 13:50:33.884 - Starting instance server at http://localhost:52427
2024-10-05 13:50:33.887 - Run workbench
> Start Eclipse e4 Workbench SWT [org.eclipse.e4.ui.workbench.swt 0.17.500.v20240807-0911]
> Start Eclipse Application Services [org.eclipse.e4.core.services 2.4.400.v20240413-1529]
> Start Eclipse UI Application Services [org.eclipse.e4.ui.services 1.6.300.v20231201-1637]
> Start Eclipse e4 Progress View [org.eclipse.e4.ui.progress 0.4.600.v20240606-1020]
> Start Eclipse Bindings Support [org.eclipse.e4.ui.bindings 0.14.400.v20240321-1245]
> Start Eclipse e4 core commands [org.eclipse.e4.core.commands 1.1.400.v20240424-0444]
> Start EMF XML/XMI Persistence [org.eclipse.emf.ecore.xmi 2.38.0.v20240721-0634]
> Start Eclipse CSS SWT Theme Support [org.eclipse.e4.ui.css.swt.theme 0.14.400.v20240424-0956]
> Start Eclipse e4 Workbench Add-ons [org.eclipse.e4.ui.workbench.addons.swt 1.5.500.v20240620-1945]
> Start Eclipse e4 Workbench SWT Renderer [org.eclipse.e4.ui.workbench.renderers.swt 0.16.500.v20240727-1037]
> Start Eclipse Dependency Injection Extensions [org.eclipse.e4.core.di.extensions 0.18.300.v20240413-1529]
> Start Eclipse CSS SWT Support [org.eclipse.e4.ui.css.swt 0.15.400.v20240321-1245]
2024-10-05 13:50:34.392 - Event loop exception
java.lang.NullPointerException: Cannot invoke "org.jkiss.dbeaver.model.app.DBPPlatform.getQueryManager()" because "org.jkiss.dbeaver.model.qm.QMUtils.application" is null
	at org.jkiss.dbeaver.model.qm.QMUtils.getDefaultHandler(QMUtils.java:62)
	at org.jkiss.dbeaver.model.runtime.features.DBRFeatureRegistry.useFeature(DBRFeatureRegistry.java:101)
	at org.jkiss.dbeaver.model.runtime.features.DBRFeature.use(DBRFeature.java:123)
	at org.jkiss.dbeaver.ui.app.standalone.ApplicationWorkbenchAdvisor.preStartup(ApplicationWorkbenchAdvisor.java:252)
	at org.eclipse.ui.internal.Workbench$20.runWithException(Workbench.java:1757)
	at org.eclipse.ui.internal.StartupThreading$StartupRunnable.run(StartupThreading.java:35)
	at org.eclipse.swt.widgets.Synchronizer.syncExec(Synchronizer.java:183)
	at org.eclipse.ui.internal.UISynchronizer.syncExec(UISynchronizer.java:133)
	at org.eclipse.swt.widgets.Display.syncExec(Display.java:4855)
	at org.eclipse.ui.internal.StartupThreading.runWithoutExceptions(StartupThreading.java:93)
	at org.eclipse.ui.internal.Workbench.init(Workbench.java:1753)
	at org.eclipse.ui.internal.Workbench.runUI(Workbench.java:2825)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:632)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:339)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:546)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:173)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.start(DBeaverApplication.java:293)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:208)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:143)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:109)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:439)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:271)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.invokeFramework(DBeaverLauncher.java:650)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.basicRun(DBeaverLauncher.java:610)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.run(DBeaverLauncher.java:1461)
2024-10-05 13:50:34.399 - DBeaver is stopping
2024-10-05 13:50:34.406 - Stop instance server
2024-10-05 13:50:35.416 - Instance server has been stopped
2024-10-05 13:50:35.429 - DBeaver shutdown completed

**Comment by E1izabeth** [comment]
@q2dm8 unfortunately, we don't support running on amd processors yet

**Comment by q2dm8** [comment]
@E1izabeth but I have Intel 
![image](https://github.com/user-attachments/assets/b37fc4e5-c722-4262-b47b-6088f8527749)

**Comment by E1izabeth** [comment]
I see "2024-10-05 13:50:33.638 - OS: Windows 10 10.0 (amd64)" in the provided logs. But OK.
Do you use installer or portable version?

**Comment by q2dm8** [comment]
@E1izabeth I used different: tried to execute portable, no reaction at all. I tried to download older versions - same issue. Now I have reisntalled windows, first thing I've done once windows installed - download and install dbeaver with installer, and it started to work. I closed and reopened and it works. I have no idea what was the problem, imho to reinstall windows to run dbeaver is quite a hardcore. Also I have checked the log, it shows amd . Don't pay attention to time, I haven't configured it yet
![image](https://github.com/user-attachments/assets/ceb27bfa-aee4-4947-853d-e9eabb2b2f32)
